ubi8-openjdk-21.yaml

```diff
@@ -0,0 +1,56 @@
+# This is an Image descriptor for Cekit
+
+schema_version: 1
+
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
+name: &name "ubi8/openjdk-21"
+version: &version "1.18"
+description: "Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK 21"
+
+labels:
+- name: "io.k8s.description"
+  value: "Platform for building and running plain Java applications (fat-jar and flat classpath)"
+- name: "io.k8s.display-name"
+  value: "Java Applications"
+- name: "io.openshift.tags"
+  value: "builder,java"
+- name: "maintainer"
+  value: "Red Hat OpenJDK <openjdk@redhat.com>"
+- name: "com.redhat.component"
+  value: "openjdk-21-ubi8-container"
+- name: "usage"
+  value: "https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/html/red_hat_java_s2i_for_openshift/"
+- name: "com.redhat.license_terms"
+  value: "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI"
+
+envs:
+- name: PATH
+  value: $PATH:"/usr/local/s2i"
+- name: "JBOSS_IMAGE_NAME"
+  value: *name
+- name: "JBOSS_IMAGE_VERSION"
+  value: *version
+- name: "LANG"
+  value: "C.utf8"
+
+ports:
+- value: 8080
+- value: 8443
+
+modules:
+  repositories:
+  - path: modules
+  install:
+  - name: jboss.container.util.pkg-update
+  - name: jboss.container.openjdk.jdk
+    version: "21"
+  - name: jboss.container.maven
+    version: "3.8.17"
```

Comment:
`3.8.21`?

Reply:
Good catch. At the moment there is no `maven-openjdk21` RPM package, which would be what `3.8.21` should install (once the module exists), but we expect there to be in the future.

Reply:
OK.

Reply:
@jmtd is there a ticket to track the creation of maven-openjdk21?

Reply:
There is for RHEL 9.4 (RHEL-13046), but not (yet) for 8.10. I asked the filer of the former whether they'd like me to clone for the latter and haven't yet had a response.